logger/logger.go

```diff
@@ -0,0 +1,69 @@
+package logger
```

Comment:
```suggestion
package log
```
"log" is in sync with other logging packages